capture_hdr_loop.py

```diff
@@ -9,60 +7,47 @@
 def _main():
     app = zivid.Application()
 
-    print("Connecting to the camera")
+    print("Connect to the camera")
     camera = app.connect_camera()
+    # camera = app.create_file_camera("C:\\Program Files\\Zivid\\data\\MiscObjects.zdf")
 
-    print("Configuring the camera settings")
+    print("Configure the camera settings")
     settings_collection = [camera.settings for _ in range(3)]
 
-    settings_collection[0].brightness = 1.8
-    settings_collection[0].gain = 1
-    settings_collection[0].filters.gaussian.enabled = True
-    settings_collection[0].filters.gaussian.sigma = 1.5
-    settings_collection[0].filters.reflection.enabled = True
+    for i in range(3):
+        settings_collection[i].brightness = 1.8
+        settings_collection[i].gain = 1
+        settings_collection[i].filters.gaussian.enabled = True
+        settings_collection[i].filters.gaussian.sigma = 1.5
+        settings_collection[i].filters.reflection.enabled = True
 
-    settings_collection[1].brightness = 1.8
-    settings_collection[1].gain = 1
-    settings_collection[1].filters.gaussian.enabled = True
-    settings_collection[1].filters.gaussian.sigma = 1.5
-    settings_collection[1].filters.reflection.enabled = True
-
-    settings_collection[2].brightness = 1.8
-    settings_collection[2].gain = 1
-    settings_collection[2].filters.gaussian.enabled = True
-    settings_collection[2].filters.gaussian.sigma = 1.5
-    settings_collection[2].filters.reflection.enabled = True
-
-    exposure_time_frame_1 = [
-        datetime.timedelta(microseconds=10000),
-        datetime.timedelta(microseconds=90000),
-        datetime.timedelta(microseconds=40000),
-    ]
-    exposure_time_frame_2 = [
-        datetime.timedelta(microseconds=40000),
+    exposure_time_frame = (
         datetime.timedelta(microseconds=10000),
-        datetime.timedelta(microseconds=90000),
-    ]
-    exposure_time_frame_3 = [
-        datetime.timedelta(microseconds=90000),
-        datetime.timedelta(microseconds=40000),
-        datetime.timedelta(microseconds=10000),
-    ]
-    iris_frames = [10, 20, 30]
+        datetime.timedelta(microseconds=15000),
+        datetime.timedelta(microseconds=20000),
+    )
+
+    iris_frames = (21, 17, 14)
 
     for i in range(len(iris_frames)):
-        settings_collection[0].exposure_time = exposure_time_frame_1[i]
-        settings_collection[1].exposure_time = exposure_time_frame_2[i]
-        settings_collection[2].exposure_time = exposure_time_frame_3[i]
+        settings_collection[0].exposure_time = exposure_time_frame[0] * (i + 1)
```

Comment:
I like the implementation of iterating through settings!
Curious to see the resulting images. We should check that the settings seem decent for an M camera at approx 1m for a scene with white black and colored objects.